db/migrate/20160321160404_add_middleware_event_streams.rb

```diff
@@ -0,0 +1,7 @@
+class AddMiddlewareEventStreams < ActiveRecord::Migration[5.0]
+  def change
+    add_column :event_streams, :middleware_type, :string   # relevant type (entity class name)
+    add_column :event_streams, :middleware_id, :bigint     # MIQ entity id
```

Comment:
And by extension middleware_name should be middleware_server_name.  That would also clarify @abonas' [comment below](https://github.com/ManageIQ/manageiq/pull/7969#discussion_r59984245)